libbeat/docs/processors-using.asciidoc

```diff
@@ -727,6 +727,16 @@ forget metadata for a container, 60s by default.
 
 beta[]
 
+[source,yaml]
+-------------------------------------------------------------------------------
+processors:
+- add_host_metadata:
+    netinfo.enabled: false
+
```

Comment:
Here the end of the source block seems to be missing.

Reply:
Should be fixed now